api.js

```diff
@@ -160,16 +161,17 @@ class Api extends EventEmitter {
 					this._setupTimeout(runStatus);
 				}
 
-				let overwatch;
+				let concurrency = Math.max(2, os.cpus().length || 1);
```

Comment:
Could you add a comment explaining the logic here? Like, why use `2` on single-core machines?

When would `os.cpus()` return an empty array?